src/components/Header/Menu/menu.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import React from 'react';
+
+function Menu() {
+  return (
+    <DivStyled>
+        <ButtonStyled>Home</ButtonStyled>
+        <ButtonStyled>Orders</ButtonStyled>
+        <ButtonStyled>FAQ</ButtonStyled>
```

Comment:
Use `.map()`

`const menuItems = ['Home', 'Orders', 'FAQ']`

and then

`menuItems.map(item => (<ButtonStyled>{item}</ButtonStyled>))`